controller/documentcontroller.php

```diff
@@ -547,6 +547,9 @@ public function wopiCheckFileInfo($fileId){
 			return false;
 		}
 
+		$instanceId = \OCP\Config::getSystemValue('instanceid', '');
+		$instanceId = substr(md5($instanceId), -13); // Don't expose the secret ID.
```

Comment:
Secret is stored separately in the config file under 'secret' key. I don't understand the comment - "Dont expose the secret ID" here. We are just calculating a md5 hash of instanceId and calling the last 13 characters as secret ID ? Can't we just expose the secret ID directly here ? Please correct me if I am missing something.